themes/black-purple-2025/layouts/_partials/disqus.html

```
@@ -0,0 +1,17 @@
+{{- $shortname := site.Params.disqusShortname -}}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)